packages/storybook/stories/Overview.stories.mdx

```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
-| [ChatComposite](./?path=/story/composites-chat-basicexample--basic-example) | Chat experience where user can send and receive messages. Thread events like typing, reads, participants entering and leaving are displayed to the user as part of the chat thread.                                                                                                                          |
+### Call composite
+
+[CallComposite](./?path=/story/composites-call-basicexample--basic-example) is providing a calling experience that allows users to start or join a call. Inside the experience users can configure their
+devices, participate in the call with video and see other participants, including those with video turn on.
+
+Here are visual examples of Call composite where participants have their camera turned off, then some ahve them turned on.
+People speaking with microphone unmuted will have their tile appearing with a blue border.
+
+<Stack
+  horizontal
+  horizontalAlign="space-between"
+  tokens={{ childrenGap: '2rem' }}
+  style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}
+>
+  <Stack.Item align="center">
+    <img
+      src="images/callComposite_CamerasOff.png"
+      alt="Call composite where all participants have their camera turned off"
+    />
+  </Stack.Item>
+  <Stack.Item align="center">
+    <img
+      src="images/callComposite_CamerasOn.png"
+      alt="Call composite where some participants have their camera turned on"
+    />
+  </Stack.Item>
+</Stack>
+
+For Teams Interop the experience includes lobby functionality for user to wait to be admitted.
+
+<Stack
+  horizontal
+  horizontalAlign="space-between"
+  tokens={{ childrenGap: '3rem' }}
+  style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}
+>
+  <Stack.Item align="center">
+    <img
+      src="images/callComposite_TeamsInterop_WaitingLobby.png"
+      alt="Call composite waiting to be admitted in a Teams meeting"
+    />
+  </Stack.Item>
+  <Stack.Item align="center" tokens={{ childrenGap: '2rem' }}>
+    <p style={{ textAlign: 'center' }}>
+      {'Popup appearing in Teams to admit ACS participant with name "I\'m joing from ACS"'}
+    </p>
+    <img
+      src="images/callComposite_TeamsInterop_AdmissionFromTeams.png"
+      alt="Popup in Teams to admit ACS participants"
+    />
+  </Stack.Item>
+</Stack>
+
+### Chat composite
+
+[ChatComposite](./?path=/story/composites-chat-basicexample--basic-example) is providing achat experience where user can send and receive
+messages. Thread events like typing, reads, participants entering and leaving are displayed to the user as part of the chat thread.
+
+Here are visual examples of Chat composite with or without participant list.
```

Comment:
As participant list is in beta (and won't be in the GA package) we should probably remove it from these screenshots